tools/src/Transforms.ts

```diff
@@ -41,19 +41,10 @@ export function transformString(
 
 async function getTransformedFileContentAsync(
   filePath: string,
-  transforms: FileTransform[],
-  options?: {
-    // File path to match the `transforms.paths` pattern, e.g. use relative path here
-    transformMatchPath?: string;
-  }
+  transforms: FileTransform[]
 ): Promise<string> {
-  // Filter out transforms that don't match paths patterns.
-  const sourceFile = options?.transformMatchPath ?? filePath;
-  const filteredContentTransforms = transforms.filter(
-    ({ paths }) =>
-      !paths ||
-      arrayize(paths).some((pattern) => minimatch(sourceFile, pattern, { matchBase: true }))
-  );
+  // Filter out transforms that don't match path patterns.
+  const filteredContentTransforms = getFilteredContentTransforms(transforms, filePath);
```

Comment:
should we remove this line as the filteredContentTransforms be moved out as a dedicated function, and both `copyFileWithTransformsAsync` and `transformFilesAsync` have direct calls to it?

Reply:
Oh, you're right 😅